deployments/Dockerfile

```
@@ -30,13 +30,20 @@ RUN --mount=type=cache,target=/go/pkg/mod \
 # and rely on host-side ownership matching UID 65532.
-RUN mkdir -p /app/data /app/pipes && chown -R 65532:65532 /app
+#
+# /app/settings is the baked seed settings directory (every key at its
+# default, the same files `wavehouse init-settings` writes) so a bare
+# `docker run` boots; production bind-mounts its own directory over it.
+# Deliberately NOT a VOLUME (unlike /app/data): it's config, not state, and an
+# anonymous volume would hide a forgotten mount.
+RUN mkdir -p /app/data /app/pipes && cp -r /src/internal/settings/seed /app/settings && chown -R 65532:65532 /app
 
 # Stage 2: Final minimal image
 FROM gcr.io/distroless/static-debian12
```

Hadolint (2.15.1)

[warning] Always tag the version of an image explicitly

(DL3006)